aws-api-appsync/src/main/java/com/amplifyframework/api/graphql/GsonResponseAdapters.java

```diff
@@ -201,6 +201,10 @@ private JsonElement skipQueryLevel(JsonElement jsonData) throws JsonParseExcepti
                 }
             }
 
+            if (message == null) {
```

Comment:
Should we also check for when the message is not null but empty?

Reply:
We might be overwriting an intentional empty message in that case. 
This particular null check is to satisfy `NonNull` expectation of Error object and avoid NPE